### PR TITLE
JSON.NET dependency bump up

### PR DESCRIPTION
**Pull request by Scooletz**
Connects to #209

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/azure-maintainers

File: src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj [hunk 1]
```diff
@@ -83,8 +83,8 @@
       <HintPath>..\packages\Mono.Cecil.0.9.6.4\lib\net45\Mono.Cecil.Rocks.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (SeanFeldman) [inline src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj]: Noticed that `AcceptanceTests` is on 10.0.2, but this one (`NServiceBus.AzureStorageQueues.Tests`) and `Tests` is on 10.0.0 for some reason.

Comment (Scooletz) [inline src/Tests/NServiceBus.AzureStorageQueues.Tests.csproj]: As far as I remember, JSON.NET differentiates between the file version and the library version. Effectively any JSON.NET 10.*.* will be visible as 10 in here.

**Comment by Scooletz** [comment]
I updated the redirects to 10 in both projects (just in case). Ping @SeanFeldman

**Pull request merged**